lib/sensu-plugin/utils.rb

```diff
@@ -61,6 +61,17 @@ def deep_merge(hash_one, hash_two)
         end
         merged
       end
+
+      def cast_bool_values_int(value)
+        case value
+        when 'true'
```

Comment:
to make it a bit more generic we should have `when 'true', true` as I am sure different implementations will return it differently.